Source/MonoGame.Extended.Tiled/TiledMapTilesetReader.cs

```diff
@@ -0,0 +1,154 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+using MonoGame.Extended.Tiled;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace MonoGame.Extended.Tiled
+{
+	public class TiledMapTilesetReader : ContentTypeReader<TiledMapTileset>
+	{
+		protected override TiledMapTileset Read(ContentReader reader, TiledMapTileset existingInstance)
+		{
+			if (existingInstance != null)
+				return existingInstance;
+
+			return ReadTileset(reader);
+		}
+
+		public static TiledMapTileset ReadTileset(ContentReader reader)
+		{
+			var texture = reader.ReadObject<Texture2D>();
```

Comment:
Guys, I need some help here... When loading the textures, there are some weird transparency things going on... Transparent black (`Color(0, 0, 0, 0)`) is rendering properly, but transparent white (`Color(255, 255, 255, 0`) is rendering the white. I'm not too experienced with the rendering system yet to know where even to look for where this inconsistency in alpha rendering is coming from (or even if it is our problem or MonoGame's). Anyone have any idea?

Reply:
This should help you: https://microsoft.github.io/Win2D/html/PremultipliedAlpha.htm

Specifically this sentence:

> In this world, RGB and alpha are linked. To make an object transparent you must reduce both its RGB (to contribute less color) and also its alpha (to obscure less of whatever is behind it). Fully transparent objects no longer have any color at all, so there is only one value that represents 100% transparency: RGB and alpha all zero.

Reply:
I completely agree, but these aren't my assets 😛
What doesn't make sense is the texture transparency works as expected when using `Content.Load<Texture2D>()` and `ContentProcessorContext.BuildAsset<Texture2DContent>()` but not `ContentProcessorContext.BuildAndLoadAsset<Texture2DContent>()`. I have tried to pass `"PremultiplyAlpha"` to the processor for the texture, but that doesn't seem to help.